### PR TITLE
HTML-CSS-Popup

### PR DESCRIPTION
**Pull request by beta-version-profile**
### HTML-CSS-Popup

[Demo](http://fierce-robin.surge.sh ) |
[Code base](https://github.com/beta-version-profile/frontend-2019-homeworks/tree/popup/submissions/beta-version-profile/popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/beta-version-profile/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                <li class="nav__item nav-menu">
+                    <input type="checkbox" class="nav__checkbox visually-hidden" id="nav__checkbox" />
+                    <label for="nav__checkbox" class="nav__label">
+                        <img src="images/menu.svg" class="nav__img" alt="menu-icon">
+                    </label>
+                    <div class="nav__wrapper">
+                        <ul class="popup">
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img src="./images/drive.svg" class="popup__icon"></img>
```
Comment (A-Ostrovnyy) [inline submissions/beta-version-profile/popup/index.html]: Add and fill `alt` attribute of `` tag.

File: submissions/beta-version-profile/popup/css/main.css [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+.popup-more__checkbox {
+  position: absolute;
+  width: 0;
+  height: 0;
+}
+
+.nav__checkbox:focus + .nav__label {
+  outline: auto;
+  outline-color: #aeb5ff;
+
```
Comment (A-Ostrovnyy) [inline submissions/beta-version-profile/popup/css/main.css]: Remove unused lines of code.

File: submissions/beta-version-profile/popup/index.html (hunk 1 above)
Comment (beta-version-profile) [reply submissions/beta-version-profile/popup/index.html]: Thx. I improved my code.

File: submissions/beta-version-profile/popup/css/main.css (hunk 2 above)
Comment (beta-version-profile) [reply submissions/beta-version-profile/popup/css/main.css]: Thx.

**Comment by A-Ostrovnyy** [comment]
Well done!

**Pull request merged**